### DISCUSSION TITLE
How to show that a note continues from previous bar in guitar tab?

### DISCUSSION THREAD

**Discussion by tmr232** [musescore/MuseScore#20697] [Q&A]
When a note continues from the previous bar, the "simple" guitar tab view does not show it at all:
![image](https://github.com/musescore/MuseScore/assets/1386239/03bbb603-227e-481a-87cd-ac562b164e08)

This makes the tab harder to read, as it seems that the next note is played right at the start of the bar.

In Guitar Pro (which I imported the tab from), we get parentheses around the relevant notes, letting the player know that they are continues from the previous bar.
![image](https://github.com/musescore/MuseScore/assets/1386239/29ee0250-42cb-4aec-a5ab-703a384e6de4)

Is there a way to achieve this in MuseScore? Preferably automatically.

Thank you!

**Comment by ghost** [comment]
You're in luck, new options for this are coming in 4.2.1! (shortly after the new year)

MuseScore will repeat the parenthesised fretmarks at the start of a new _system_ already, but not at the start of a new bar otherwise; that option is coming. For now, to show tied notes in tab, your only option is to go to Stave/Part properties > Advanced style properties... and turn on 'Show back-tied fret marks', which will show the ties and repeat the fretmarks. That certainly makes the intention clear, but is perhaps more than you want.

**Reply by tmr232** [reply]
Thanks for the quick reply!

As for "Show back-tied fret marks" - it does make things a bit too busy for what I want. But yes, it's better than not having that.

As for the new options - is there a build / issue / branch I can try out?

**Reply by ghost** [reply]
Not yet. Watch this space
